deep_speech_2/audio_data_utils.py

```diff
@@ -258,18 +284,14 @@ def instance_reader_creator(self,
 
         :param manifest_path: Filepath of manifest for audio clip files.
         :type manifest_path: basestring
-        :param sort_by_duration: Sort the audio clips by duration if set True
-                                 (for SortaGrad).
-        :type sort_by_duration: bool
+        :param sortagrad: Sort the audio clips by duration in the first epoc
+                          if set True (for SortaGrad).
+        :type sortagrad: bool
         :param shuffle: Shuffle the audio clips if set True.
```

Comment:
Since it is not a thorough instance-wise shuffle, but a specific batch-wise shuffle. Could you add more explanation to users?
Besides, rename shuffle to batch_shuffle?